readme.md

```diff
@@ -135,6 +135,37 @@ const tasks = new Listr([
 It's also possible to return a `stream`. The stream will be converted to an `Observable` and handled as such.
 
 
+#### Skipping tasks
+
+<img src="media/skipped.png" width="255" align="right">
+
+At any point during execution a task can be skipped by invoking the `skip()` function on the task instance passed in to the task function.
+
+```js
+const tasks = new Listr([
+	{
+		title: 'Task 1',
+		task: () => Promise.resolve('Foo')
+	},
+	{
+		title: 'Can be skipped',
+		task: task => {
+			if (someCondition) {
+				task.skip('Reason for skipping');
```

Comment:
My initial implementation involved something similar and had to be returned but I thought that might make it trickier to skip tasks that use streams/observables (I'm not super familiar with either so that may just be my ignorance and perhaps its no big deal?).

Throwing is super easy and its a nice side effect that the remaining code isn't run but you're right it is a little obtuse... If you think returning a symbol is the better option I'm happy to make the change.

Also there is a comment (2 lines down) but perhaps its not clear enough?